dashel/dashel-common.cpp

```diff
@@ -148,9 +132,9 @@ namespace Dashel
 			{
 				address = INADDR_ANY;
 			}
-#else // WIN32
+#else  // WIN32
```

Comment:
Why the extra space?

Reply:
no idea :)

Reply:
It's an option : SpacesBeforeTrailingComments